assets/js/src/ajax.js

```diff
@@ -56,6 +56,10 @@ MailPoet.Ajax = {
       data: this.options.data || {},
     };
   },
+  constructGETUrl: function constructGETUrl(options) {
```

Comment:
Not important but nicer version of camel case is `constructGetUrl` - more readable, unambiguous.